modules/hooks/init.js

```diff
@@ -667,6 +667,19 @@ export default function() {
       type: Boolean
     });
 
+    game.settings.register("wfrp4e", "templateCollisionMethod", {
+      name: `SETTINGS.templateCollisionMethod`,
+      hint: `SETTINGS.templateCollisionMethodHint`,
+      scope: 'world',
+      config: true,
+      type: String,
+      choices: {
+        "centerPoint": "SETTINGS.templateCollisionCenterPoint",
+        "grid": "SETTINGS.templateCollisionGrid",
+        "area": "SETTINGS.templateCollisionArea"
+      },
+  });
+
```

Comment:
My gut reaction here is that it's a bit much, the system provides the basic functionality of area effects, then modules can provide more specific alternatives if desired.

Reply:
that would be tricky to handle at this point, unless some methods of AreaHelper could be easily overwritten. If this is no-go, i can think about something else and leave only the token center implementation.